Sources/ApolloCodegenLib/ApolloCodegen.swift

```diff
@@ -266,6 +304,67 @@ public class ApolloCodegen {
 
     try SchemaModuleFileGenerator.generate(config, fileManager: fileManager)
   }
+
+  private static func findExistingGeneratedFilePaths(
+    config: ConfigurationContext,
+    fileManager: ApolloFileManager = .default
+  ) throws -> Set<String> {
+    var globs: [Glob] = []
+    globs.append(Glob(
+      ["\(config.output.schemaTypes.path)/**/*.graphql.swift"],
+      relativeTo: config.rootURL
+    ))
+
+    switch config.output.operations {
+    case .inSchemaModule: break
+
+    case let .absolute(operationsPath):
+      globs.append(Glob(
+        ["\(operationsPath)/**/*.graphql.swift"],
+        relativeTo: config.rootURL
+      ))
+
+    case let .relative(subpath):
+      let searchPaths = config.input.operationSearchPaths.map { searchPath -> String in
+        let startOfLastPathComponent = searchPath.lastIndex(of: "/") ?? searchPath.firstIndex(of: ".")!
```

Comment:
I'm assuming we trust in using `!` here because of the new validation on input search paths?

Reply:
That's right!